pages/_app.tsx

```diff
@@ -0,0 +1,8 @@
+import { AppProps } from 'next/app';
+import '../css/index.css';
```

Comment:
This applies the styles globally which can be hard to maintain. 
Instead, try using something like CSS in JS or Add component level CSS.

Check this [link](https://nextjs.org/docs/basic-features/built-in-css-support) for more info